app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss

```diff
@@ -34,21 +34,36 @@
     text-align: center;
   }
 
-  &__radio-btn--centered{
-    padding: 0 0 21px 0;
-
-    &:before{
-      top: 100%;
-      margin-top: -35px;
-      left: 50%;
-      margin-left: -12px;
+  &__radio-btn--centered {
+
+    input[type="radio"] + label{
+      padding: 0 0 46px 0;
+
+      &:before{
```

Comment:
the single : is from CSS2 so I think we're trying to move over to :: which is CSS3.